Sources/Automerge/Document.swift

```diff
@@ -768,6 +768,61 @@ public final class Document: @unchecked Sendable {
         }
     }
 
+    /// Get the list of marks for a text object at ``Cursor``.
```

Comment:
Using ``Cursor`` symbol as the first line (the abstract) links in an awkward way in the DocC output. I'd recommend something more like:

```suggestion
    /// Get the list of marks inside a text object at the position you indicate with a cursor.
```

And link to cursor in the content of the reference, if you want to do that.